src/main/java/hu/oe/nik/szfmv/environment/models/Road.java

```diff
@@ -10,12 +91,52 @@
      */
     public Road(int x, int y, String imageFileName) {
```

Comment:
Az adattagok után a konstruktoroknek kellene állnia, sima metódus nem előzheti meg, tehát a hashMapInit-et a konstruktorok után kellene mozgatni (Javában nem kell a használat előtt deklarálni a metódusokat).

Reply:
Okés, köszi, nem igazán volt tiszta, hogy a "Constructor definition in wrong order" mire is vonatkozik pontosan. :D